infracost-usage-example.yml

```
@@ -92,10 +95,7 @@ resource_usage:
   aws_docdb_cluster_instance.my_db:
     data_storage_gb: 1000         # Total storage for cluster in GB.
     monthly_io_request: 100000000 # Monthly number of input/output requests for cluster.
-    monthly_cpu_credit_hours: 100 # Monthly CPU credits used over the instance baseline in vCPU-hours, only applicable for T3 instances. 
-    
-  aws_docdb_cluster.my_cluster:
-    backup_storage_gb: 10000      # Amount of backup storage that is in excess of 100% of the storage size for the cluster in GB.
+    monthly_cpu_credit_hours: 100 # Monthly CPU credits used over the instance baseline in vCPU-hours, only applicable for T3 instances.
```

```suggestion
    monthly_cpu_credit_hrs: 100 # Monthly CPU credits used over the instance baseline in vCPU-hours, only applicable for T3 instances.
```

(following the contributing guide's "Usage file guidelines" > units section). Same with the following 2 occurrences of this key. Should this and the following use a similar description as `monthly_cpu_credit_hrs` above?